squad/core/models.py

```diff
@@ -46,9 +46,11 @@ def accessible_to(self, user):
         projects = Project.objects.accessible_to(user)
         project_ids = [p.id for p in projects]
         group_ids = set([p.group_id for p in projects])
+        if not isinstance(user, AnonymousUser):
+            group_ids = group_ids | set([g.id for g in Group.objects.filter(members__in=[user])])
         return self.filter(
-            Q(id__in=group_ids) | Q(members__id=user.id)
-        ).order_by('slug').annotate(
+            id__in=group_ids
+        ).distinct().order_by('slug').annotate(
```

Comment:
I tried it locally, `distinct()` is unnecessary as you already removed the join `members__id=user.id`.